payments/models.py

```diff
@@ -668,17 +668,18 @@ def update_lease_and_application(self, new_status):
         self.lease.save(update_fields=["status", "comment"])
 
         if new_status == OrderStatus.PAID:
-            application = self.lease.application
-            application.status = ApplicationStatus.HANDLED
-            self.lease.application.save(update_fields=["status"])
-
-            if (
-                isinstance(self.lease, WinterStorageLease)
-                and application.area_type == ApplicationAreaType.UNMARKED
-            ):
-                sticker_number = get_next_sticker_number(self.lease.start_date)
-                self.lease.sticker_number = sticker_number
-                self.lease.save(update_fields=["sticker_number"])
+            if self.lease.application:
```

Comment:
If you like, add a comment that missing application might happen due to TIMMI imported data (and any other situations where an order might exist without application?)